plugins/woocommerce/legacy/css/twenty-twenty-two.scss

```diff
@@ -26,6 +26,7 @@
 }
 
 @import "mixins";
+@import "animation";
```

Comment:
Point of interest: this isn't used in the Twenty Twenty One stylesheet, as it is hinging on a similar keyframe definition that is supplied by Blocks.